db/migrate/20200225104037_add_social_preview_template_to_tags.rb

```diff
@@ -0,0 +1,5 @@
+class AddSocialPreviewTemplateToTags < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tags, :social_preview_template, :string, default: "article"
```

Comment:
Since we will be looking up by this field a lot we should probably add an index for it.

Reply:
I assumed that it's not a lot, but ok :+1: